src/tensors/gpu/cuda_helpers.h

```diff
@@ -4,7 +4,7 @@
 #include "3rd_party/exception.h"
 #include "common/logging.h"
 
-const float CUDA_FLT_MAX = 1.70141e+38;
+__constant__ float CUDA_FLT_MAX = 1.70141e+38;
```

Comment:
What is this?

Reply:
Would you mind adding a note as a very brief line-end comment? E.g. // fixes a missing constant in CUDA-side headers